Userland/Libraries/LibGUI/AbstractThemePreview.h

```diff
@@ -24,7 +24,7 @@ class AbstractThemePreview : public GUI::Frame {
 
     Gfx::Palette const& preview_palette() const { return m_preview_palette; }
     void set_preview_palette(Gfx::Palette const&);
-    ErrorOr<void> set_theme_from_file(Core::File&);
+    ErrorOr<void> set_theme_from_file(DeprecatedString const& path, NonnullOwnPtr<Core::Stream::File>);
```

Comment:
Just wondering, can this be a `StringView` rather than a `DeprecatedString const&`?

Reply:
*Probably.*
I didn't use a StringView initially to avoid string allocation when calling `on_theme_load_from_file()` but looks like nobody even sets this function anymore. *do you think we can remove it? >:))*

Reply:
I guess so! I don't know why it (apparently I) added it :P